package/gluon-status-page/src/build.js

```diff
@@ -1,6 +1,7 @@
 ({
   paths: {
-    "bacon": "../Bacon"
+    "bacon": "../Bacon",
+    "eventsource": "../EventSource"
```

Comment:
Referencing it here won't help, require.js will only include code that is referenced. I think it would make sense to just download the pre-minified version from the git repo, copy that into the output dir in Package/gluon-status-page/install, and reference it in the HTML code like you did before.